### ISSUE TITLE
Galas with movement ring from E1 S3

### ISSUE THREAD

**Issue by Konrad22** [project-ethea/Invasion_from_the_Unknown#32]
Galas starts the first turn (in probably any scenario) with as many MP as he would have without the ring.

**Comment by shikadiqueen** [comment]
What version of Wesnoth are you running? It seems to work just fine for me in 1.14.2.

**Comment by Konrad22** [comment]
I'm not sure if it was 1.14.1 or 1.14.2 (it updated yesterday at some time when I was stll playing), but I can confirm that the problem still exists in 1.14.2.
I can also specify it, this happens so far only in S4. Everyting is fine in S5.
![galas](https://user-images.githubusercontent.com/39015360/40583624-dc897f64-6192-11e8-8441-f5b344f19930.png)
![galas2](https://user-images.githubusercontent.com/39015360/40583637-fcefc862-6192-11e8-8c98-f1bb390dd76c.png)

**Comment by shikadiqueen** [comment]
Yep, confirmed. Have yet to see if it's a bug in the scenario code (for the previous scenario — the start-of-scenario save for Over the Sands already has Galas set to 6/7 moves in the snapshot) or a bug in the engine.

**Comment by Konrad22** [comment]
The same thing happens in [S8 An Errand of Hope].
